src/main/java/org/elasticsearch/index/mapper/internal/VersionFieldMapper.java

```diff
@@ -96,7 +96,7 @@ protected Field initialValue() {
     };
 
     public VersionFieldMapper(Settings indexSettings) {
-        super(new Names(NAME, NAME, NAME, NAME), Defaults.BOOST, Defaults.FIELD_TYPE, null, null, null, null, null, null, indexSettings);
+        super(new Names(NAME, NAME, NAME, NAME), Defaults.BOOST, Defaults.FIELD_TYPE, false, null, null, null, null, null, indexSettings);
```

Comment:
I don't think that would have helped in this case, since `toXContent()` here just returns the builder. But yes, it should be `true` and it is now fixed.